exponent_server_sdk/__init__.py

```diff
@@ -277,7 +277,7 @@ class PushClient(object):
     DEFAULT_MAX_MESSAGE_COUNT = 100
     DEFAULT_MAX_RECEIPT_COUNT = 1000
 
-    def __init__(self, host=None, api_url=None, **kwargs):
+    def __init__(self, host=None, api_url=None, session, **kwargs):
```

Comment:
This arg should be optional?

```suggestion
    def __init__(self, host=None, api_url=None, session=None, **kwargs):
```

Could also do with a docstring update to mention what it's for